label_studio/utils/argparser.py

```diff
@@ -172,4 +177,25 @@ def valid_filepath(filepath):
         args.label_config = os.path.join(find_dir('examples'), args.template, 'config.xml')
     if not hasattr(args, 'label_config'):
         args.label_config = None
+
+    # The following part merges with existing arguments if provided.
+    # Command-line arguments have priority over existing arguments.
+    if existing_args:
+        # We need to filter for arguments that were actually provided in the
+        # command line - otherwise we overwrite with default values:
+        args_set = {key:value for key, value in vars(args).items() if value}
+
+        # Merge these dicts: First use the complete list with defaults (args),
+        # then overwrite with the existing arguments, then overwrite these 
+        # with the arguments that were actually set (args_set):
+        args_merged = {**vars(args), **vars(existing_args), **args_set}
+        args = SimpleNamespace(**args_merged)
+
+    # To comply with legacy code, some arguments have to be set as default
+    # if they aren't defined yet:
+    args.root_dir = args.root_dir or "."
+    args.input_format = args.input_format or "json"
+    args.sampling = args.sampling or "sequential"
```

Comment:
Defaults are now listed here and set to "None" above. This is because we need to differentiate between user-provided arguments and arguments that are set by default.

According to convention, the priority should be: (With cmd line the highest)
cmd line -> config file -> defaults